### PR TITLE
Added Maintenance Configurations to Event Grid System Topic Toc

### PR DESCRIPTION
**Pull request by AyushKSinghal**

### CODE REVIEW

File: articles/virtual-machines/maintenance-configurations-how-to-event-grid.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+title: Send events from Azure API Management to Event Grid
```
Comment (ApnaLakshay) [inline articles/virtual-machines/maintenance-configurations-how-to-event-grid.md:2]: Replace "Azure API Management" with "Maintenance Configuration"

File: articles/virtual-machines/maintenance-configurations-how-to-event-grid.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+# Send events from Maintenance Configuration to Event Grid
+
+Maintenance Configuration integrates with [Azure Event Grid](../event-grid/overview.md) so that you can send event notifications to other services and trigger downstream processes. Event Grid is a fully managed event routing service that uses a publish-subscribe model. Event Grid has built-in support for Azure services like [Azure Functions](../azure-functions/functions-overview.md) and [Azure Logic Apps](../logic-apps/logic-apps-overview.md), and can deliver event alerts to non-Azure services using webhooks.
+
+For example, using integration with Event Grid, you can build an application that start or shutdown VM, and sends an email notification each time maintenance schedule start or completes.
+
+In this article, you subscribe to Event Grid events for a Maintenance Configuration, trigger events, and send the events to an endpoint that processes the data. To keep it simple, you send events to a sample storage queue that show message:
+
+:::image type="content" source="media/how-to-event-grid/event-grid-viewer-intro.png" alt-text="API Management events in Event Grid viewer":::
+
```
Comment (ApnaLakshay) [inline articles/virtual-machines/maintenance-configurations-how-to-event-grid.md:21]: Is this a new file added here? If yes, then I can't see the file in the commit

File: articles/virtual-machines/maintenance-configurations-how-to-event-grid.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+[!INCLUDE [azure-cli-prepare-your-environment.md](../../includes/azure-cli-prepare-your-environment.md)]
+- If you don't already have an API Management service, complete the following quickstart: [Create an Azure API Management instance](get-started-create-service-instance.md)
+- Enable a [system-assigned managed identity](api-management-howto-use-managed-service-identity.md#create-a-system-assigned-managed-identity) in your API Management instance.
```
Comment (ApnaLakshay) [inline articles/virtual-machines/maintenance-configurations-how-to-event-grid.md:24]: @AyushKSinghal Make sure to remove all the API Management references in this document

File: articles/virtual-machines/maintenance-configurations-how-to-event-grid.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+- Create a [resource group](../azure-resource-manager/management/manage-resource-groups-portal.md#create-resource-groups) if you don't have one in which to deploy the sample endpoint.
+
+## Create an event endpoint
+
+In this section, you use a Resource Manager template to deploy a pre-built sample web application to Azure App Service. Later, you subscribe to your API Management instance's Event Grid events and specify this app as the endpoint to which the events are sent.
```
Comment (ApnaLakshay) [inline articles/virtual-machines/maintenance-configurations-how-to-event-grid.md:29]: Remove/Replace the API Management reference here

File: articles/virtual-machines/maintenance-configurations-how-to-event-grid.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+Once the deployment has succeeded (it might take a few minutes), open a browser and navigate to your web app to make sure it's running:
+
+`https://<your-site-name>.azurewebsites.net`
+
+You should see the sample app rendered with no event messages displayed.
+
+[!INCLUDE [event-grid-register-provider-portal.md](../../includes/event-grid-register-provider-portal.md)]
+
+## Subscribe to API Management events
```
Comment (ApnaLakshay) [inline articles/virtual-machines/maintenance-configurations-how-to-event-grid.md:56]: Please check if this section is relevant for maintenance configuration. If no, then remove this section

File: articles/virtual-machines/maintenance-configurations-how-to-event-grid.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+1. On the **Basic** tab:
+    * Enter a descriptive **Name** for the event subscription.
+    * In **Event Types**, select one or more API Management event types to send to Event Grid. For the example in this article, select at least **Microsoft.APIManagement.ProductCreated** 
+    * In **Endpoint Details**, select the **Web Hook** event type, click **Select an endpoint**, and enter your web app URL followed by `api/updates`. Example: `https://myapp.azurewebsites.net/api/updates`.
+    * Select **Confirm selection**.
+1. Leave the settings on the remaining tabs at their default values, and then select **Create**.
+
+    :::image type="content" source="media/how-to-event-grid/create-event-subscription.png" alt-text="Create an event subscription in Azure portal":::
+
+## Trigger and view events
```
Comment (ApnaLakshay) [inline articles/virtual-machines/maintenance-configurations-how-to-event-grid.md:71]: Remove/Replace the API Management references here

File: articles/virtual-machines/maintenance-configurations-how-to-event-grid.md (hunk 1 above)
Comment (AyushKSinghal) [reply articles/virtual-machines/maintenance-configurations-how-to-event-grid.md:2]: work in progress, will complete by today

**Pull request merged**